### PR TITLE
RHODS-2212: Updated cards according to microcopy recommendations.

### PR DESCRIPTION
**Pull request by dlabaj**
Updated cards according to microcopy recommendations.

The following are screenshots of the changes:

Screen Shot 2021-12-07 at 4 28 46 PM
Screen Shot 2021-12-07 at 4 33 40 PM
Screen Shot 2021-12-07 at 4 33 33 PM
Screen Shot 2021-12-07 at 4 33 24 PM
Screen Shot 2021-12-07 at 4 33 56 PM
Screen Shot 2021-12-07 at 4 35 21 PM

- [x] The commits are squashed in a cohesive manner and have meaningful messages.
- [x] For commits that came from upstream, `[UPSTREAM]` has been prepended to the commit message
- [x] JIRA link(s):  https://issues.redhat.com/browse/RHODS-2212 & https://issues.redhat.com/browse/RHODS-2220
- [x] The Jira story is acked
- [x] An entry has been added to the latest build document in [Build Announcements Folder](https://drive.google.com/drive/folders/1sgkK1WZgGo9CXsLizNe0GbAzVKuSKrZL).
- [x] Testing instructions have been added in the PR body (for PRs involving changes that are not immediately obvious)

### CODE REVIEW

**Review by abigaeljamie — APPROVED** [review]

File: frontend/src/__tests__/__snapshots__/EnabledApplications.spec.tsx.snap [hunk 1]
```diff
@@ -12,5 +12,5 @@ exports[`EnabledApplications should display a message when there are no enabled
...
...pported-image\\" src=\\"../images/CheckStar.svg\\" alt=\\"Red Hat certified and supported\\"></span></div><div class=\\"pf-c-card__body\\"><div class=\\"odh-card__partner-badge-container\\"><span class=\\"odh-card__partner-badge\\">Red Hat managed</span></div>A multi-user version of the notebook designed for companies, classrooms and research labs.</div><div class=\\"pf-c-card__footer odh-card__footer\\"><a class=\\"odh-card__footer__link\\" href=\\"https://jupyterhub-redhat-ods-applications.apps.jephilli-4-9-06-21-0953.devcluster.openshift.com\\" target=\\"_blank\\" rel=\\"noopener noreferrer\\">Launch application<svg style=\\"vertical-align: -0.125em;\\" fill=\\"currentColor\\" height=\\"1em\\" width=\\"1em\\" viewBox=\\"0 0 512 512\\" aria-hidden=\\"true\\" role=\\"img\\"><path d=\\"M432,320H400a16,16,0,0,0-16,16V448H64V128H208a16,16,0,0,0,16-16V80a16,16,0,0,0-16-16H48A48,48,0,0,0,0,112V464a48,48,0,0,0,48,48H400a48,48,0,0,0,48-48V336A16,16,0,0,0,432,320ZM488,0h-128c-21.37,0-32.05,25.91-17,41l35.73,35.73L135,320.37a24,24,0,0,0,0,34L157.67,377a24,24,0,0,0,34,0L435.28,133.32,471,169c15,15,41,4.5,41-17V24A24,24,0,0,0,488,0Z\\"></path></svg></a></div></article></div></section></div>"
```
Comment (jkoehler-redhat) [inline frontend/src/__tests__/__snapshots__/EnabledApplications.spec.tsx.snap:15]: It is really hard to tell what is changed due to length of line.  Not sure if this has to be all on one line.  I realize you are just updating, but wanted to point this out as it is hard to review (and I am sure to know what to change).

**Review by jkoehler-redhat — COMMENTED** [review]
From the changes that I feel qualified to review, these look fine to me.

File: frontend/src/__tests__/__snapshots__/EnabledApplications.spec.tsx.snap (hunk 1 above)
Comment (dlabaj) [reply frontend/src/__tests__/__snapshots__/EnabledApplications.spec.tsx.snap:15]: This file is generated by jest test suite.  Unfortunately, I can't modify this file by hand.

**Review by crobby — CHANGES_REQUESTED** [review]
Looks like this needs a rebase.

**Comment by dlabaj** [comment]
@crobby Rebased is complete should be good now.

**Review by crobby — APPROVED** [review]
built, tested, approved.  /lgtm

**Pull request merged**